requirements/requirements.txt

```diff
@@ -73,6 +73,7 @@ git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e2
 Pygments==2.0.2
 tinys3==0.1.11
 django-formtools==2.0
+django-otp==0.3.13
```

Comment:
That was the main reason, but after looking at it I also noticed django-two-factor-auth specifies >=0.3.4,<0.99, which means prod environments could have very different versions even if there aren't functional differences. 0.3.13 is the latest release